src/panels/config/automation/trigger/types/ha-automation-trigger-event.ts

```diff
@@ -38,6 +38,14 @@ export class HaEventTrigger extends LitElement implements TriggerElement {
         .defaultValue=${event_data}
         @value-changed=${this._dataChanged}
       ></ha-yaml-editor>
+      <ha-yaml-editor
```

Comment:
We shouldn't just put up a YAML editor. Instead we know that context ID can only have a user that is relevant for matching, so we should show a drop down where the user can pick users to match.